### DISCUSSION TITLE
When will the DryIoc.dll will be updated to the latest 5.0.2 version and release the package?

### DISCUSSION THREAD

**Discussion by Anoop1001** [PrismLibrary/Prism#2690] [Xamarin.Forms] [answered]
When will the DryIoc.dll will be updated to the latest 5.0.2 version and release the package?

I have an external dependency on DryIoc.dll v5.0.2 and the version I am using is 4.7.3 as we are consuming Prism.DryIoc.Extensions. 

I have also posted the problem on [Stackoverflow](https://stackoverflow.com/q/72257891/7198636)

**Comment by brianlagunas** [comment]
This is a project we work on in our spare time, and we release when a number of factors have aligned. Right now we are focused on leaving the .NET Foundation which should be official by the end of this month. So a release will definitely not happen before then.

**Comment by dansiegel** [comment, marked as answer]
The most telling part here is that you think you have an issue while you're using Prism.DryIoc.Extensions. That package is meant to REPLACE the official Prism DI package. you should instead be using Prism.Forms directly and remove the Prism.DryIoc.Forms reference. You will lose PrismApplication and have to instead inherit from PrismApplicationBase. This will force you to implement `CreateContainerExtension` for which you'll just return the static instance of the container from Prism.DryIoc.Extensions.

**Reply by dansiegel** [reply]
Also note that we no longer monitor Stack Overflow as we feel it has become very toxic.

**Reply by Anoop1001** [reply]
Thank you very much for your response. Now the issue is resolved.

Prism.DryIoc.Forms is using DryIoc.dll in DryIocContainerExtension. I cloned the repo, updated DryIoc.dll to 5.0.2 and built it, Found that WithoutFastExpressionCompiler has been removed in DryIoc.dll latest(v5.0.2). So removed the method and created a dll and started using in my project. It is now working fine.

**Reply by sigmarsson** [reply]
> Also note that we no longer monitor Stack Overflow as we feel it has become very toxic.

@dansiegel How exactly do you mean this ? I don't understand how objective facts could become toxic.

**Reply by dansiegel** [reply]
Stack Overflow moderators are toxic. Their behavior has caused several OSS Maintainers including the Prism team to stop using their platform.
